README.md

```diff
@@ -1,4 +1,4 @@
-Dash Core staging tree 18.0
```

Comment:
let's not do this here; please do a new PR that removes the version from this file

Reply:
we have version here because out 'master' is a release, but bitcoin doesn't have it because its master is development branch and they cares only about tags;
that's why its here.... 

Let me drop it